src/test/java/uk/gov/pay/api/utils/mocks/ChargeResponseFromConnector.java

```diff
@@ -0,0 +1,198 @@
+package uk.gov.pay.api.utils.mocks;
+
+import uk.gov.pay.api.model.CardDetails;
+import uk.gov.pay.api.model.PaymentState;
+import uk.gov.pay.api.model.RefundSummary;
+import uk.gov.pay.api.model.SettlementSummary;
+import uk.gov.pay.commons.model.SupportedLanguage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+
+public class ChargeResponseFromConnector {
+    public final Long amount, corporateCardSurcharge, totalAmount; 
```

Comment:
I get that this is only within test but I think we agreed no public immutable fields.

https://docs.google.com/document/d/1iWzWIsitOUdlyBZGqWzT49R27OqHidxl8Kh5-FMcrSY/edit

Reply:
yup good call, fixed.